src/selector/extra.js

```diff
@@ -143,3 +143,13 @@ export const siblings = function(selector) {
 export const slice = function(start, end) { // eslint-disable-line no-unused-vars
   return $([].slice.apply(this, arguments));
 };
+
+export const concat = function( elements ) {
+  const nodes = this;
+  each( elements, element => {
+    if( nodes.indexOf( element ) === -1 ) {
```

Comment:
The jquery-compat mode has an `index` method (not `indexOf`), like jQuery itself; so you can't use this directly. Perhaps something like `[].indexOf.call(this, element)` is better here.